openid-caep-specification-1_0.md

```diff
@@ -691,6 +706,56 @@ NOTE: The event type URI is wrapped, the backslash is the continuation character
 ~~~
 {: #device-compliance-change-examples-out-of-compliance title="Example: Device No Longer Compliant - Complex Subject + optional claims"}
 
+## Session Established {#session-established}
+Event Type URI:
+
+`https://schemas.openid.net/secevent/caep/event-type/session-established`
+
+The Session Established event signifies that the Transmitter has created a new session for a user for use with the Transmitter. Receivers may use this information for a number of reasons, including:
```

Comment:
I think we may need to avoid saying that the "Transmitter has created a new session for a user for use with the Transmitter" because in many cases it will be a different entity that is paired with a transmitter (e.g. Transmitter and IdP are different logical components). We were generic with the text in other events (ex: "Token Claims Change signals that a claim in a token, identified by the subject claim, has changed.")